docs/http/traffic-policy/actions/jwt-validation.mdx

```diff
@@ -56,56 +61,56 @@ present in the request. If all tokens are not present a `401` status code will b
 
 | Parameter    | Type                               | Description                  |
 | ------------ | ---------------------------------- | ---------------------------- |
-| `allow_list` | [JWTIssuer](#jwtissuer-parameters) | the list of allowed issuers. |
+| `allow_list` | [JWTIssuer](#jwtissuer-parameters) | The list of allowed issuers. |
```

Comment:
The convention here is lowercase. I don't feel strongly about this, just saying that's what it is everywhere else.

Reply:
fine :)